t/35-script_clone_job.t

```diff
@@ -222,24 +219,25 @@ subtest 'get 2 nodes HA cluster with get_deps' => sub {
 
 subtest 'error handling' => sub {
     my $tx = Mojo::Transaction->new;
-    my $test = sub { OpenQA::Script::CloneJob::handle_tx($tx, 42, {name => 'testjob'}, 'base-url', {}, {}, 0, []) };
+    my %url_handler = (local_url => Mojo::URL->new('https://base-url/foo/bar'));
+    my $test = sub { OpenQA::Script::CloneJob::handle_tx($tx, \%url_handler, {}, {42 => {name => 'testjob'}}) };
     throws_ok { $test->() } qr/Failed to create job, empty response/, 'empty response handled';
     $tx->res->body('foobar');
     throws_ok { $test->() } qr/Failed to create job, server replied: "foobar"/, 'unexpected non-JSON handled';
     $tx->res->code(200)->body('{"foo": "bar"}');
     throws_ok { $test->() } qr/Failed to create job, server replied: \{ foo => "bar" \}/, 'unexpected JSON handled';
-    ($tx = Mojo::Transaction->new)->res->code(200)->body('{"id": 43}');
-    combined_like { $test->() } qr|Created job #43: testjob -> base-url/t43|, 'expected response';
+    ($tx = Mojo::Transaction->new)->res->code(200)->body('{"ids": {"42": 43}}');
+    combined_like { $test->() } qr|Created job #43: testjob -> https://base-url/t43|, 'expected response';
```

Comment:
the output changed here only because of how you use the URL differently within the test, so normally we wouldn't see a difference, right?

Reply:
Yes, it simply uses the `--host` URL here. Previously the test's fake values didn't include a scheme but now it does so now the test expectation actually looks more like the output would look in reality.